jicoco-kotlin/src/main/kotlin/org/jitsi/config/JitsiConfig.kt

```diff
@@ -17,38 +17,42 @@
 package org.jitsi.config
 
 import com.typesafe.config.ConfigFactory
-import org.jitsi.utils.config.ConfigSource
-import org.jitsi.utils.logging2.LoggerImpl
+import org.jitsi.metaconfig.ConfigSource
+import org.jitsi.service.configuration.ConfigurationService
 
 /**
- * Creates and holds the [ConfigSource] instances for the legacy and new
- * config files.
+ * Holds common [ConfigSource] instances for retrieving configuration.
+ *
+ * Should be renamed to JitsiConfig once the old one is removed.
  */
+@Suppress("unused", "MemberVisibilityCanBePrivate")
 class JitsiConfig {
     companion object {
-        private val logger = LoggerImpl(JitsiConfig::class.qualifiedName)
-        val newConfig: ConfigSource = JitsiConfigFactory.newConfigSupplier()
-        val legacyConfig: ConfigSource = JitsiConfigFactory.legacyConfigSupplier()
-        @JvmStatic
-        val legacyConfigShim = JitsiConfigFactory.legacyConfigurationServiceShimSupplier()
+        /**
+         * A [ConfigSource] loaded via [ConfigFactory].
+         */
+        val TypesafeConfig: ConfigSource = TypesafeConfigSource("typesafe config", ConfigFactory.load())
```

Comment:
Is it a convention that the names of `val`s in the companion object start with a capital letter?

Reply:
I took this style from the kotlin guys from how they defined [Dispatchers](https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-core/jvm/src/Dispatchers.kt#L33)...I think it's more differentiating the default value.  I don't feel strongly though.